documentation/header.php

```diff
@@ -8,6 +10,12 @@
         $pagetitle => substr($_SERVER['REQUEST_URI'], 1) // remove the starting '/'
     );
     require('../../header.php');
+    
+    if($version != $latest)
+    {
+        $linklatest = '<div class="important"><strong>Warning:</strong> this page refers to an old version of SFML. Click <a href="../' . $latest . '/' . str_replace('documentation/' . $version . '/', '', $breadcrumbs[$pagetitle]) . '">here</a> to switch to the latest version.</div>';
+    }
 ?>
 
-<h1>Documentation of SFML <?php echo $version ?></h1>
+<h1>Documentation of SFML <?php echo $version; ?></h1>
```

Comment:
I've no idea if it's relevant or not, but on the same line in the french header there's no semicolon...